discord/ext/commands/formatter.py

```diff
@@ -142,11 +142,31 @@ class HelpFormatter:
     width: :class:`int`
         The maximum number of characters that fit in a line.
         Defaults to 80.
+    ending_note 
+        The ending note modify strings that is at end of help page. Useful for i18n.
+        This property could either be a plain string or a callable that takes
+        :class:`.HelpFormatter` as its parameter and returns :class:`str`.
+        Defaults to a callable like
+
+        .. code-block:: python
+
+            async def _ending_note_default(formatter):
+                command_name = formatter.context.invoked_with
+                return "Type {0}{1} command for more info on a command.\\n" \\
+                    "You can also type {0}{1} category for more info on a category." \\
+                    .format(formatter.clean_prefix, command_name)
+
+    commands_heading: :class:`str`
+        The command list's heading string used when the help command is invoked with a categoly name.
+        Useful for i18n. Defaults to ``"Commands:"``
     """
-    def __init__(self, show_hidden=False, show_check_failure=False, width=80):
+    def __init__(self, show_hidden=False, show_check_failure=False, width=80,
+                 ending_note=None, commands_heading="Commands:"):
         self.width = width
         self.show_hidden = show_hidden
         self.show_check_failure = show_check_failure
+        self.ending_note = ending_note
```

Comment:
I feel like this shouldn't be an attribute but something that should be changed through inheritance and subclassing. This is why it's in a separate method, so you can do something like the following:

```py
class MyHelpFormatter(commands.HelpFormatter):
    def get_ending_node(self):
      return '...'
```

I do agree with the `commands_heading` though so I think that's good.